### PR TITLE
Add in some helpful text

### LINKED ISSUE #569: Add "Not sure where to start?" to WelcomePage
* Generate a resource with `lucky gen.resource.browser Posts title:String body:String`
* Generate a new action with `lucky gen.action.browser`
* etc.

### PR DESCRIPTION
**Pull request by paulcsmith**
Closes #569 

screen shot 2018-08-24 at 4 42 49 pm

### CODE REVIEW

File: src/lucky/welcome_page.cr [hunk 1]
```diff
@@ -41,12 +38,87 @@ class Lucky::WelcomePage
...
+    end
+  end
+
+  def render_help
+    h1 "Not sure where to start? Here are some ideas:", class: "headline"
+
+    table class: "help-table" do
+      tbody do
+        tr do
+          td "Change the page this URL renders", class: "left-column"
```
Comment (edwardloveall) [inline src/lucky/welcome_page.cr]: What do you think about

> Edit this page in: `src/actions/home/index.cr`

Comment (paulcsmith) [reply src/lucky/welcome_page.cr]: Hmm, you're not actually changing the page though, your changing the page it renders or the action it redirects to. Open to other ideas though. I'm not super sold on what's there!

Comment (edwardloveall) [reply src/lucky/welcome_page.cr]: Hmm action is sort of a tough concept to get across. Maybe

> Change the backend of this page

Comment (paulcsmith) [reply src/lucky/welcome_page.cr]: That seems a bit misleading to me. Almost like you'll change the whole framework :P
 
Maybe `Change the what this page renders`? So you don't really need to start off knowing about actions and all that?

Comment (edwardloveall) [reply src/lucky/welcome_page.cr]: Hah! `Pull requests welcome` 😉 

I like what you put but

```diff
-Change the what this page renders
+Change what this page renders
```

Comment (paulcsmith) [reply src/lucky/welcome_page.cr]: Nah...I like the typo better 😎

**Pull request merged**